tsconfig.json

```diff
@@ -1,19 +1,10 @@
 {
   "compilerOptions": {
-    "rootDir": "src",
-    "outDir": "dist",
-    "module": "ES2015",
-    "target": "es2017",
-    "lib": ["es6"],
+    "module": "nodenext",
+    "target": "esnext",
+    "lib": ["es6", "dom"],
```

Comment:
Why is the dom lib being added?

Also you can extend one of the standard node tsconfigs: https://www.npmjs.com/package/@tsconfig/node16

Reply:
The reason for `dom` is that `module-from-string` depends on `esbuild`, and `esbuild`'s types contain a reference to `WebAssemby`:

```
> serverless-rollup-plugin@1.1.0 build
> tsc -p tsconfig.dist.json

node_modules/esbuild/lib/main.d.ts:584:16 - error TS2503: Cannot find namespace 'WebAssembly'.

584   wasmModule?: WebAssembly.Module
                   ~~~~~~~~~~~


Found 1 error in node_modules/esbuild/lib/main.d.ts:584
```
This definition is available in the `dom` lib, so I added it for the build to succeed. An alternative approach is to set `skipLibChecks` in `tsconfig.json`, I can really go either way here...